client/src/account_address.rs

```diff
@@ -0,0 +1,89 @@
+use std::{fs, str};
+
+use clap::{App, Arg, ArgMatches, SubCommand};
+
+use casper_types::{AsymmetricType, PublicKey};
+
+use crate::{command::ClientCommand, common};
+
+/// This struct defines the order in which the args are shown for this subcommand's help message.
+enum DisplayOrder {
+    Verbose,
+    Key,
+}
+
+/// Handles providing the arg for and retrieval of the key.
+mod key {
+    use casper_node::crypto::AsymmetricKeyExt;
+    use casper_types::AsymmetricType;
+
+    use super::*;
+
+    const ARG_NAME: &str = "public-key";
+    const ARG_SHORT: &str = "pk";
+    const ARG_VALUE_NAME: &str = "FORMATTED STRING or PATH";
+    const ARG_HELP: &str = "Path to the public key or the formatted hex string";
```

Comment:
This should be way more helpful.  It should be more like the help string for the `query-state` subcommand, as specified in the task description.

Reply:
Changed description to more closely reflect that in `query-state`